sass/styles.scss

```diff
@@ -1 +1,547 @@
-p {color:red;}
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed:300,400,700|Roboto:100,300,400,500,700');
+
+/*variables*/
+$color-accent: #cc071e;
+$color-body: #353535;
+$body-font: Roboto, sans-serif;
+
+// $link: #000000;
+// $visit: #cc071e;
+// $hover: #cc071e;
+// $active: #000000;
+
+/*mixins*/
+@mixin link($link, $hover, $active) {
+  a {
+    text-decoration:none;
+    color: $link;
+      &:hover {
+        color: $hover;
+        text-decoration:underline;
+      }
+      &:active {
+        color: $active;
+      }
+  }
+}
+
+$html-margin: 0px auto !default;
+$padding: 0px !default;
+$font-size: 62.5% !default;
+
+
+@include link(black, #cc071e,black);
+
+
+body {
+    font-family: $body-font;
+    font-size:1.2em;
+    margin:0px;
+    padding:0px;
+}
+
+
+
+.detail-list {
+  padding-left:20px;
+    li {
+      list-style-position:inside;
+      list-style-type:circle;
+      font-weight:300;
+      margin-left:20px;
+  }
+}
+
+.detail-small {
+    width:100px;
+    margin-bottom:50px;
+}
+
+.features {
+    margin-bottom:70px;
+}
+
+form {
+  margin-top:60px;
+    p {
+      float:left;
+      display:block;
+      height:25px;
+  }
+  &:after {
+   content: "";
+   display: table;
+   clear: both;
+ }
+}
+
+html {
+    margin:0px auto;
+    width:320px;
+    padding:0px;
+    font-size:62.5%;
+}
+
+header {
+  margin-top:10px;
+    img {
+      height:70px;
+      margin-left:90px;
+      margin-bottom:30px;
+    }
+}
+
+h1 {
+    display:none;
+}
+
+h2 {
```

Comment:
Watch for and remove redundancy/repetition. For example, this setting appears here and in two other media queries with no changes. Reduce to the one necessary setting. Look for other similar situations and remove them.

Reply:
Hint: Use the Find feature in Atom to make this quick and easy to look through your code for same selectors and compare to reduce repetition.